src/main/java/com/baller/application/service/ClubService.java

```diff
@@ -97,4 +103,40 @@ public void applyClub(Long memberId, Long clubId) {
         clubJoinApplyMapper.createClubJoinApply(ClubJoinApply.ofRequested(memberId, clubId));
     }
 
+    @RequireClubRole({ClubRoleType.LEADER})
+    public List<ClubJoinApplyResponse> getClubJoinApply(Long clubId) {
```

Comment:
```suggestion
    public List<ClubJoinApplyResponse> getClubApplies(Long clubId) {
```
이정도로 해도 되지 않을까요? 사전적으로 join과 apply가 좀 유사하게 느껴지긴하네요 (참여지원??)

Reply:
나머지도 네이밍 다시 봐주세요 :)